services/masterdata/src/lib/components/masterdata-type-details/masterdata-type-details.component.ts

```diff
@@ -47,11 +49,9 @@ export class MasterdataTypeDetailsComponent implements OnInit {
 	}
 
 	prepareForDisplay(item: MasterdataType) {
-		return Object.entries(item).map(item => {
-			item[0] = item[0].toLowerCase();
-			if (item[0] === "masterdatascount") {
-				item[0] = `masterdatas ${item[0].replace("masterdatas", "")}`;
-			}
+		// "metadata" model field will have a special display!
+		return Object.entries(item).filter(item => item[0] !== "metadata").map(item => {
+			item[0] = this.translateService.instant(`masterdatamgmt.pages.masterdataTypeDetails.modelFields.${item[0]}`);
 			return item;
```

Comment:
Unreachable code due to duplicate return statement.

Reply:
@Johanh-od I don't understand the issue here?